moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -842,18 +846,18 @@
                </size>
               </property>
               <property name="toolTip">
-               <string>Allows the robot to replan if the plan to be executed collides with the environment.</string>
+               <string>Allow controlling the interactive marker through ROS topics (under /rviz/moveit/marker/ ) published other nodes.</string>
```

Comment:
See my previous comment. Triggering planning and execution becomes also possible.
Not sure though, this is a useful feature. If I want to control the robot from an app, I would directly talk to the move_group node instead...

Reply:
I thought I changed that. I must have forgotten to save. Will fix tomorrow.

Reply:
Users of this feature reported in github issues before. I agree this is not ideal, but I guess it can simplify User-Machine-Interaction pipelines.

Reply:
I figured it could be useful for some interface, even if it seems a bit strange. I updated the comment.